src/runtime.js

```diff
@@ -55,6 +55,9 @@ import {installCidService} from './service/cid-impl';
 import {installCryptoService} from './service/crypto-impl';
 import {installDocumentInfoServiceForDoc} from './service/document-info-impl';
 import {installDocumentStateService} from './service/document-state';
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.